pkg/abstractions/image/image.go

```diff
@@ -139,3 +158,17 @@ func convertBuildSteps(buildSteps []*pb.BuildStep) []BuildStep {
 	}
 	return steps
 }
+
+func (is RuncImageService) getBaseImageTag(pythonVersion string, micromambaEnabled bool) (string, bool) {
```

Comment:
feels weird to parameterize the function like this, is there a more elegant way we can map to the tags?

Reply:
In cleaning up the above, I got rid of this